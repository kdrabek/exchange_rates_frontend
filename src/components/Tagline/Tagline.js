import React from 'react';
import { Row, Col, PageHeader } from 'react-bootstrap';

const Tagline = (props) => {
  return (
    <Row>
      <Col xs={12} md={12}>
        <PageHeader>
          Kursy walut NBP <br /><small>Wed≈Çug tabeli A</small>
        </PageHeader>
      </Col>
    </Row>
  );
};

export default Tagline;
