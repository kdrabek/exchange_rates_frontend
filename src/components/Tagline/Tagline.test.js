import React from 'react';
import ReactDOM from 'react-dom';
import Tagline from './Tagline';
import renderer from 'react-test-renderer';

test('Tagline component matches the snapshot', () => {
  const component = renderer.create(
    <Tagline />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});