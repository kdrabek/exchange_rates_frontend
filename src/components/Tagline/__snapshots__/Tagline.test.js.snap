exports[`test Tagline component matches the snapshot 1`] = `
<div
  className="row">
  <div
    className="col-md-12 col-xs-12">
    <div
      className="page-header">
      <h1>
        Kursy walut NBP 
        <br />
        <small>
          Wed≈Çug tabeli A
        </small>
      </h1>
    </div>
  </div>
</div>
`;
